apps/extension/src/background/ports/completion.ts

```diff
@@ -57,7 +51,7 @@ const handler: PlasmoMessaging.PortHandler<
 
   const predictedModel = await _getCompletionModel(config, txn)
   if (!isOk(predictedModel)) {
-    _maybeInterrupt(id, predictedModel)
+    Extension._maybeInterrupt(id, predictedModel)
```

Comment:
Public methods called from other classes should not have underscores on them. The name of the method is also less clear now that it doesn't exist in this file.

Reflecting on it further, Extension might not be the best place for this logic because it's mainly focused on abstracting away platform-specific code for writing extensions, and there's some app specific logic here. Maybe one of the utils files is better? Any ideas?

Reply:
b/l/helpers.ts

Reply:
done, refactored out